packages/opentelemetry-exporter-collector-grpc/README.md

```diff
@@ -27,7 +27,7 @@ const { BasicTracerProvider, SimpleSpanProcessor } = require('@opentelemetry/tra
 const { CollectorTraceExporter } =  require('@opentelemetry/exporter-collector-grpc');
 
 const collectorOptions = {
-  // url is optional and can be omitted - default is localhost:4317
+  // url is optional and can be omitted - default is grpc://localhost:4317
   url: '<collector-hostname>:<port>',
```

Comment:
Here and elsewhere, this might be warranted
```suggestion
  url: '<collector-scheme>://<collector-hostname>:<port>',
```

Reply:
But, for this specific case, where the collector is grpc only, shouldn't we inform the use the required scheme is `grpc`? Or does it actually support other schemes?

Reply:
then it should be `'grpc://<collector-hostname>:<port>'` right ?